packages/nimble-blazor/NimbleBlazor/Components/NimbleLabelProviderCore.razor.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.AspNetCore.Components;
+
+namespace NimbleBlazor;
+
+/// <summary>
+/// Core Label Provider for Nimble
+/// </summary>
+public partial class NimbleLabelProviderCore : ComponentBase
+{
+    [Parameter]
+    public string? PopupDismiss { get; set; }
+
+    [Parameter]
+    public string? NumericIncrement { get; set; }
+
+    [Parameter]
+    public string? NumericDecrement { get; set; }
+
+    /// <summary>
+    /// Gets or sets a collection of additional attributes that will be applied to the created element.
+    /// </summary>
+    [Parameter(CaptureUnmatchedValues = true)]
+    public IReadOnlyDictionary<string, object>? AdditionalAttributes { get; set; }
```

Comment:
Not sure if we should just skip adding the `AdditionalAttributes` junk. This component certainly doesn't seem like it will benefit from the use of arbitrary attributes (plus the fact that I believe we want to ultimately move away from this API for all components).

Reply:
Removed

Reply:
@atmgrifter00 for consistency based from the last discussion I was advocating for putting AdditionalAttributes on everything: https://github.com/ni/nimble/pull/1231#discussion_r1189089284
My understanding is that is the current direction. The reasoning being these are web components, not blazor components, and have additional properties that are valid.

This does keep coming up, so if we want to reverse that decision and opt-in on only some components adding AdditionalAttributes configuration then we can. In that case I would like a way to see easily in the component that it was decided that it was opted out, not that it was accidentally forgotten.

Reply:
Our current conventions also state to always include it: https://github.com/ni/nimble/blob/main/packages/nimble-blazor/CONTRIBUTING.md#conventions

So if a direction change is being made that should be updated as well

Reply:
Fair points. Added it back for consistency.